src/main/java/duke/Command.java

```diff
@@ -0,0 +1,104 @@
+package duke;
+
+public class Command {
```

Comment:
According to the module's coding standards, should a Javadoc comment exist above the `public class Command`? The same command for all other classes.